kotlin-react-router-dom/src/main/generated/react/router/NavigationContext.kt

```diff
@@ -5,4 +5,5 @@
 
 package react.router
 
+@JsName("UNSAFE_NavigationContext")
 external val NavigationContext: react.Context<NavigationContextObject>
```

Comment:
it would be great to rename these context, because now it is not obvious that it is delicate API